string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,42 @@
+/* exported titleCase */
+function titleCase(string) {
+  var stringArray = string.split(' ');
+  var wordString = '';
+  var firstLetter = '';
+  var otherLetters = '';
+  for (var i = 0; i < stringArray.length; i++) {
+    wordString = stringArray[i];
+    if (stringArray[i] !== 'and' && stringArray[i] !== 'or' && stringArray[i] !== 'nor' && stringArray[i] !== 'but' && stringArray[i] !== 'a' && stringArray[i] !== 'an' && stringArray[i] !== 'the' && stringArray[i] !== 'as' && stringArray[i] !== 'at' && stringArray[i] !== 'by' && stringArray[i] !== 'for' && stringArray[i] !== 'in' && stringArray[i] !== 'of' && stringArray[i] !== 'on' && stringArray[i] !== 'per' && stringArray[i] !== 'to') {
```

Comment:
Maybe you should do this with an array of the words that are exceptions, and then determine if the word is present in that array.